docs/releases/5.1.1.txt

```diff
@@ -35,3 +35,6 @@ Bugfixes
 * Restored, following a regression in Django 5.1, the ability to override the
   timezone and role setting behavior used within the ``init_connection_state``
   method of the PostgreSQL backend (:ticket:`35688`).
+
+* Adjusted the admin fieldset template to reduce debug logging when rendering
+  (:ticket:`35716`).
```

Comment:
How about something like this (or a variation of)?
```suggestion
* Fixed a regression in Django 5.1 where logging about missing variables was
  emitted when including the admin fieldset template snippet (:ticket:`35716`).
```

Reply:
Made a tweak trying to copy the format of some of the messages in the [3.2.1 release](https://docs.djangoproject.com/en/5.1/releases/3.2.1/#bugfixes), specifically:
```
* Fixed a bug in Django 3.2 where variable lookup errors were logged rendering
  the sitemap template if alternates were not defined (:ticket:`32648`).

* Fixed a bug in Django 3.2 where variable lookup errors were logged when
  rendering some admin templates (:ticket:`32681`).
```